### DISCUSSION TITLE
Twig Intl Extension not loading

### DISCUSSION THREAD

**Discussion by beheist** [pimcore/pimcore#10544] [Q&A / Support]
Hi all,

since Pimcore X, I have the problem that the Twig Intl extension no longer loads. I followed the installation steps [here](https://twig.symfony.com/doc/3.x/filters/format_date.html), but it seems the extension is just never loaded into the Symfony environment. I even added a "die('foo');" into the Bundle class to see if it is parsed, and it is not. Is there anything specific that I have to do in addition for that bundle to load?

Thanks!

**Comment by NiklasBr** [comment]
Make sure you have added the extension with [registerBundlesToCollection()](https://pimcore.com/docs/pimcore/current/Development_Documentation/Extending_Pimcore/Bundle_Developers_Guide/Bundle_Collection.html#page_Bundle-Dependencies) in your `Kernel` class as well.

**Reply by beheist** [reply]
Thanks, that does add the extension, but it just leads me to the next error. After adding 
```
$collection->addBundle(new TwigExtraBundle());
```
the bundle now shows up when I run `./bin/console config:dump-reference`. However, when I call my controller, I now get this error message:
![image](https://user-images.githubusercontent.com/10347669/136706636-327deb39-5318-4b2c-aba3-151dfc568294.png)

No idea why it now tries to load the extension twice..

**Comment by jheimbach** [comment]
Because I googled the same problem and come to this, and there is no statisfing answer:

In Pimcore 10.6 it is already fixed with the direct integration of `twig/extra-bundle`, just require `twig/intl-extra`


In Pimcore < 10.6 you have to require and register the ExtraBundle yourself (in bundles.php or Kernel.php), but you will get the error mentioned from @beheist "StringExtension is already loaded"

For that, you can use the config:
```
twig_extra:
    string: false
```
StringExtension will be enabled regardless of this setting (through Pimcore, not through ExtraBundle) 

Link to MR that fixed this issue: https://github.com/pimcore/pimcore/pull/13418
